File: Assignments/Lessons[47-54]/A6.cpp
#include "main.h"
 
int main() {
    int i = 0, result = 10;

    cout << "Using for loop" << endl;
    cout << "================================\n";
    
    for (i = 0 ; i < 5; i++)
    {
        cout << result << endl;
        result *= 10;
    }

    cout << endl;

    i = 0;
    result = 10;

    cout << "Using while loop" << endl;
    cout << "================================\n";

    while (i < 5)
    {
        cout << result << endl;
        result *= 10;
        i++;
    }
    return 0;
}

File: Front_end_dev/JAVASCRIPT/JAVASCRIPT_Assignments/Lessons[57-63]/A6.js
function multiply(...args) {
  let result = 1
  for (arg of args) {
    if (typeof arg === 'number') {
      result *= Math.trunc(arg)
    }
  }
  console.log(result)
}

multiply(10, 20); // 200
multiply("A", 10, 30); // 300
multiply(100.5, 10, "B"); // 1000